assignment3.txt

```diff
@@ -0,0 +1,7 @@
+1. it is invalid because a for loop has 3 requirements inside the for loop. (initialisation; condition; incrementat). It does not meet these requirements.
+2. It is an infinite loop. The number will keep getting smaller.
```

Comment:
Correct! there is one more thing that would cause a compilation warning - try and find it!